pkg/server/signapi/handle_sign.go

```diff
@@ -99,9 +99,9 @@ func (sa *SignApi) HandleSign(c echo.Context) error {
 	// Signing option
 	var algo string
 	switch c.QueryParam("signing_option") {
-	case "", "ssh-rsa":
+	case "ssh-rsa":
 		algo = ssh.KeyAlgoRSA
-	case "rsa-sha2-256":
+	case "rsa-sha2-256", "":
```

Comment:
No particular opinions whether to switch the default to `rsa-sha2-256` or `rsa-sha2-512`, but I suppose there's a reason for the existence of the 256 one, and it's the next incremental step up, so chose it here.